### PR TITLE
samples: Updated README for thermometer sample

### PR DESCRIPTION
**Pull request by Jandro20**
Updated README.txt to .rst format.

This pull request is necessary for issue: https://github.com/zephyrproject-rtos/zephyr/issues/27805.

### CODE REVIEW

File: samples/sensor/thermometer/README.rst [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+A simple thermometer example, which writes the temperature to the console once per second.
+
+Building and Running
+********************
+
+It can be built and executed on bbc_microbit as follows:
+
+.. code-block:: console
+
+	make
```
Comment (yperess) [inline samples/sensor/thermometer/README.rst:18]: This doesn't seem right? Shouldn't it use `west build samples/sensor/thermometer`?

**Review by yperess — CHANGES_REQUESTED** [review]

File: samples/sensor/thermometer/README.rst (hunk 1 above)
Comment (MaureenHelm) [reply samples/sensor/thermometer/README.rst:18]: It should use `zephyr-app-commands`

Comment (Jandro20) [reply samples/sensor/thermometer/README.rst]: Thanks for the review! It's should be okay with the last modification.

**Comment by MaureenHelm** [comment]
Please address the failed compliance check

**Review by MaureenHelm — APPROVED** [review]

**Review by kartben — APPROVED** [review]

**Comment by MaureenHelm** [comment]
@yperess please revisit

**Review by yperess — APPROVED** [review]

**Pull request merged**